sdk/python/lib/pulumi/runtime/settings.py

```diff
@@ -338,6 +338,11 @@ def _sync_monitor_supports_invoke_transforms() -> bool:
         return False
     return SETTINGS.feature_support["invokeTransforms"]
 
+def _sync_monitor_supports_parameterization() -> bool:
+    if "parameterization" not in SETTINGS.feature_support:
+        return False
+    return SETTINGS.feature_support["parameterization"]
```

Comment:
Yea so we've renamed this which is good but the call site is wrong then.

Reply:
Side note -- in general, could these all be `SETTINGS.feature_support.get("<name>", False)`?